danylevichoo/second/task.1.py

```diff
@@ -0,0 +1,53 @@
+'''
+Користувач вводить рядок, що містить римський запис числа. Напишіть функцію, що повертає це число у арабському записі. Виведіть результат.
+У випадку помилкового вводу виведіть повідомлення про помилку.
+(Римські позначення цифр: M=1000, D=500, C=100, L=50, X=10, V=5, I=1. Жоден символ не може повторюватися понад 3 рази підряд.)зи підряд.)
+'''
+import re
+
+re_rom = re.compile("^[MDCLXVI]+$")
+regex = re.compile((r'^.*(.)(\1)(\1).*$'))
+n = "Y"
+while n == "Y":
+    def if_number(text):
+        for i in text:
+            if bool(re_rom.match(text)):
+
+                if bool(regex.match(text)):
+                    print("Invalid char.")
+
+                    break
+
+                else:
+                    return text
+            else:
+                print("Invalid char.")
+
+                break
+
+
+
+    print("Input romanian number:")
+    a = input()
```

Comment:
Name of variable _`a`_ can be replaced by an appropriate word _(`text, number` etc )_
Assigning a value to a variable can be replaced with one line _`a = input("Input romanian number:")`_